packages/components/src/components/ContentCard/ContentCard.tsx

```diff
@@ -103,28 +91,43 @@ export const ContentCard = ({
     },
   };
 
+  const isCardInteractive = interactiveElementType === "card";
+  const isButton = interactiveElementType === "button";
+  const inAnchor = interactiveElementType === "anchor";
+
+  const interactiveElementProps = {
+    href,
+    onClick,
+    target,
+    rel: "noopener noreferrer",
+    cursor: "pointer",
+  };
+
   return (
     <Box.div
-      as={href ? "a" : as}
+      as={isCardInteractive ? "a" : as}
       backgroundColor={backgroundColor[background]}
       border={0}
       borderRadius="borderRadius40"
-      boxShadow={{
-        hover: "shadowStrong",
-        focusWithin: "shadowStrong",
-      }}
-      cursor={href ? "pointer" : "default"}
+      boxShadow={
+        isCardInteractive
+          ? {
+              _: "none",
+              hover: "shadowStrong",
+            }
+          : "none"
+      }
```

Comment:
```suggestion
      boxShadow={
        _: "none",
        hover: isCardInteractive && "shadowStrong",
      }
```

Not sure if it works.